pages/documentation/developer/api_guide/single-line-diagram_svg-writing.md

```diff
@@ -57,22 +52,22 @@ First of all, we need to add some Maven dependencies in our `pom.xml` file:
 </dependencies>
 
 <properties>
-    <powsybl.sld.version>1.7.2</powsybl.sld.version>
-    <powsybl.core.version>3.7.1</powsybl.core.version>
+    <powsybl.sld.version>1.8.0</powsybl.sld.version>
+    <powsybl.core.version>3.8.0</powsybl.core.version>
     <slf4j.version>1.7.22</slf4j.version>
 </properties>
 ```
 
 Here are some details about these dependencies:
 - `powsybl-single-line-diagram-core` is the core module of single-line-diagram,
-- `powsybl-iidm-impl` is for the Network model,
-- `powsybl-iidm-test` is for loading the test Network,
+- `powsybl-iidm-impl` is for the network model,
+- `powsybl-iidm-test` is for loading the test network,
 - `powsybl-config-test` is for loading the test configuration (see [configuration API guide](configuration.md)),
 - `powsybl-cgmes-conversion` and `powsybl-triple-store-impl-rdf4j`  are for importing a CGMES file,
 - `slf4j-simple` allows you to have simple logging capabilities.
 
```

Comment:
Can you add a link to https://www.powsybl.org/pages/documentation/developer/artifacts.html for powsyblb artifacts ?